app/javascript/components/pages/Home.js

```diff
@@ -1,14 +1,30 @@
 
 import React, { Component } from "react"
+import Header from '../components/Header.js'
+import Footer from '../components/Footer.js'
 
 
 class Home extends Component{
 render() {
+    const {
+        logged_in,
+        sign_in_route,
+        sign_out_route,
+        new_user_route
+      } = this.props
+
```

Comment:
It doesn't look like Home component is being passed any props from App.js so these props will be undefined